src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -223,4 +225,20 @@ public void close() {
         }
     }
 
+    /**
+     * Return the server status of Rabbit MQ calculated on the number of active consumers.
+     *
+     * @return
+     */
+    public boolean isRabbitMQServerUp() {
+        boolean rabbitServerIsUp = container.getActiveConsumerCount() >= 1;
```

Comment:
Or you can implement a AMQP connection listerner that is available in Spring.
https://github.com/spring-projects/spring-amqp/blob/master/spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionListener.java

Reply:
Implementation done in next push.